modules/pattern/pattern.resolver.js

```diff
@@ -29,7 +42,11 @@ const patternMutation = {
 
   deletePattern: async (parent, args) => {
     try {
-      return await patternService.deletePattern(args.id);
+      const pattern = await Pattern.findById(args.id).lean();
```

Comment:
Please, don't call the model in the resolver, you should only call service here. It is a violation of the layered pattern

Reply:
fixed